docs/topics/multiplatform/multiplatform-advanced-project-structure.md

```diff
@@ -168,6 +168,32 @@ There are three important concepts in dependency resolution:
 
    ![Error on JVM-specific API in common code](dependency-resolution-error.png){width=700}
 
+### Aligning versions of common dependencies across source sets
+
+Common dependencies need to be aligned across source sets to make sure that common code is always compiled against
+the version of a library that satisfies all source sets.
+
+In the example above, imagine that you want to add the `androidx.navigation:navigation-compose:2.7.7` dependency to your
+`androidMain` source set. Your project explicitly declares the `kotlinx-coroutines-core:1.7.3` dependency for the `commonMain`
+source set, but the Compose Navigation library with the version 2.7.7 requires Kotlin coroutines 1.8.0 or newer.
+
+Resolving this, Gradle applies `kotlinx-coroutines-core:1.8.0` to the `commonMain` source set. But to make the common code
+compile consistently across targets, the iOS source sets also need to be constrained to the same dependency version.
```

Comment:
I'm not an english guru, just asking. 

what's the difference between `across targets` vs `across all targets` in this context?

Reply:
"targets" refers to an undetermined set of targets, and "all targets" — to a determined set (not necessarily defined explicitly).

To me "all targets" seems to implicitly refer to all KMP-supported targets, not all project targets. But looking at this fragment again, I do think it's better to clarify, for example, "across all configured targets."